components/contribution-flow/index.js

```diff
@@ -472,10 +473,14 @@ class CreateOrderPage extends React.Component {
       this.props.onSuccess();
     }
     if (this.props.redirect && this.isValidRedirect(this.props.redirect)) {
-      const orderId = get(orderCreated, 'id', null);
-      const transactionId = get(orderCreated, 'transactions[0].id', null);
-      const status = orderCreated.status;
-      const redirectTo = `${this.props.redirect}?orderId=${orderId}&transactionid=${transactionId}&status=${status}`;
+      const urlParams = new URLSearchParams({
+        orderId: get(orderCreated, 'id', null),
+        orderUuid: get(orderCreated, 'uuid', null),
+        transactionid: get(orderCreated, 'transactions[0].id', null),
+        transactionUuid: get(orderCreated, 'transactions[0].uuid', null),
```

Comment:
Not a fan of this because transactions `uuid` are considered private by the API, we should not share it with third parties.

Reply:
Thank you for pointing that. In the context of GraphQL API v2, what should we use as public identifier for transactions? Hashid of the auto-increment id, like orders?

Reply:
Hashid of the auto-increment id sounds good